js/lib/urlutil.js

```diff
@@ -520,6 +520,12 @@ const UrlUtil = {
     return ip.isPrivate(hostname) || hostname === 'localhost' || whitelistSuffixes.some((suffix) => {
       return hostname && hostname.endsWith(`.${suffix}`)
     })
+  },
+
+  shortenUrlToDomainTLD: (url) => {
+    if (url != null && (!url.startsWith('http://') && !url.startsWith('https://')) ? `https://${url}` : url) {
```

Comment:
not sure I understand this `if` condition. `!url.startsWith('https://')) ? `https://${url}` : url` returns a truthy value, but it doesn't set the URL

Reply:
i think you mean:
```
if (url != null) {
  url = (!url.startsWith('http://') && !url.startsWith('https://')) ? `https://${url}` : url
  return url.substring(0, url.indexOf('/', url.indexOf('://') + 3))
}

Reply:
Yes thank you.